src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -294,6 +321,45 @@ public Wallet LoadWallet(string password, string name)
             return wallet;
         }
 
+        /// <inheritdoc />
+        public void UnlockWallet(string password, string name, int timeout)
```

Comment:
I think we want [NoTrace] on any methods with passwords to ensure they don't get written to log file.

Reply:
indeed

Reply:
In fact, I'm wondering. The logging of passwords will only happen when the node is run with `loglevel = trace` and not in the normal run. Tracing is usually used by devs during testing. Do we really want to remove all the logging from the concerned methods?

Reply:
I'll leave this task for another day.